networkx/algorithms/tests/test_bridges.py

```diff
@@ -51,6 +53,53 @@ def test_multiedge_bridge(self):
         assert list(nx.bridges(G)) == [(2, 3)]
 
 
+class TestHasBridges:
+    """Unit tests for the has bridges function."""
+
+    def test_single_bridge(self):
+        edges = [
+            # DFS tree edges.
+            (1, 2),
+            (2, 3),
+            (3, 4),
+            (3, 5),
+            (5, 6),
```

Comment:
Not really important, but might be nice since the graph is not particularly easy to visualize (for me anyways) from the edgelist.

```suggestion
            (5, 6),  # The only bridge edge
```